docs/sample_inventories/controller_broker_scram_kraft.yml

```diff
@@ -0,0 +1,49 @@
+---
+##
+## The following is an example inventory file of the configuration required for setting up Confluent Platform with Kraft Controller with SCRAM.
+## This will set up controller-controller communication via plain and other communication via scram. Please note: The value must be added at kafka_controller to achieve this behaviour.
+
+all:
+  vars:
+    ansible_connection: ssh
+    ansible_user: ec2-user
+    ansible_become: true
+    ansible_ssh_private_key_file: /home/ec2-user/guest.pem
+    ansible_python_interpreter: /usr/bin/python3
+
+    ssl_enabled: true
+    sasl_protocol: scram
```

Comment:
also should we add 1-2 lines depecting how they can define scram users?

Reply:
This is an existing feature, there is no change to process of defining scram users.